datasafe-directory/datasafe-directory-impl/src/main/java/de/adorsys/datasafe/directory/impl/profile/config/MultiDFSConfig.java

```diff
@@ -0,0 +1,36 @@
+package de.adorsys.datasafe.directory.impl.profile.config;
+
+import de.adorsys.datasafe.encrypiton.api.types.UserID;
+import de.adorsys.datasafe.types.api.resource.AbsoluteLocation;
+import de.adorsys.datasafe.types.api.resource.BasePrivateResource;
+import de.adorsys.datasafe.types.api.resource.Uri;
+
+import java.net.URI;
+
+public class MultiDFSConfig extends DefaultDFSConfig {
+
+    private final Uri profilesPath;
+
+    public MultiDFSConfig(URI fsPath, URI profilesPath) {
+        super(fsPath, "PAZZWORT");
```

Comment:
"PAZZWORT" actually comes from configuration, it is declared in DatasafeProperties